### DISCUSSION TITLE
Is my Github student's pack declined?

### DISCUSSION THREAD

**Discussion by Aarya-Gosar** [community/community#22843] [New to GitHub]
<p>So I have been waiting for the student’s pack for 15 days ( I don’t mind the long wait, considering that they have 1000s of applications.)</p>
<p>I just want to know, if my application is declined, then would it show “it has been declined” or would they ignore it and keep it as “pending”.</p>
<p>I would prefer the first one as I don’t want to be waiting for nothing.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>You will be told if your application is accepted or declined. Please remember, this is the absolute busiest time of year for our Education team, and I would expect you may be waiting another couple weeks. Your application is definitely still pending if that’s what it says.</p>
